src/main/java/com/employee/controller/EmployeeController.java

```diff
@@ -11,134 +12,200 @@
 public class EmployeeController {
 
 	private EmployeeService employeeService;
-	
+
 	public EmployeeController(EmployeeService employeeService) {
-		this.employeeService = employeeService; 
+		this.employeeService = employeeService;
 	}
+
 	public static void main(String[] args) {
-		
+
 		EmployeeService employeeService = new EmployeeServiceImpl();
 		EmployeeController employeeController = new EmployeeController(employeeService);
+
+		// employeeController.addEmployee();
+
+		// employeeController.findAllEmployee();
+
+		// employeeController.findEmployeeByLastName("Kim");
+
+		// employeeController.updateEmployee();
+		//employeeController.deleteEmployee();
 		
-	        //employeeController.addEmployee();
-	        
-	       // employeeController.findAllEmployee();
-		
-		//employeeController.findEmployeeByLastName("Kim");
-		
-		//employeeController.updateEmployee();
-		employeeController.deleteEmployee();
+		employeeController.addEmployeeBatch();
+	}
+
+	public void addEmployeeBatch() {
+		List<Employee> employeesToInsert = new ArrayList<>();
+
+		Employee Alice = new Employee();
```

Comment:
@hninpwintaung-hpa object name should start with small letter. Not Alice, it's alice.